### PR TITLE
AIS-46 | @jdwjdwjdw | Resolve all content-related Level A accessibility issues on the Decanter V6 website

### PR DESCRIPTION
**Pull request by jdwjdwjdw**
# NOT READY FOR REVIEW
- (Edit the above to reflect status)

# Summary
- [AIS-46](https://stanfordits.atlassian.net/browse/AIS-46): Resolve all content-related Level A accessibility issues on the Decanter V6 website

# Review By (Date)
- When does this need to be reviewed by?

# Criticality
- How critical is this PR on a 1-10 scale? Also see [Severity Assessment](https://stanfordits.atlassian.net/browse/D8CORE-1705).
- E.g., it affects one site, or every site and product?

# Review Tasks

## Setup tasks and/or behavior to test

1. Check out this branch
2. Navigate to...
3. Verify...

## Front End Validation
- [ ] Is the markup using the appropriate semantic tags and passes HTML validation?
- [ ] Cross-browser testing has been performed?
- [ ] Automated accessibility scans performed?
- [ ] Manual accessibility tests performed?
- [ ] Design is approved by @ user?

### Code
- [ ] Are the naming conventions following our standards?
- [ ] Does the code have sufficient inline comments?
- [ ] Is there anything in this code that would be hidden or hard to discover through the UI?
- [ ] Are there any [code smells](https://blog.codinghorror.com/code-smells/)?
- [ ] Are tests provided? eg (unit, behat, or codeception)

### Code security
- [ ] Are all [forms properly sanitized](https://www.drupal.org/docs/8/security/drupal-8-sanitizing-output)?
- [ ] Any obvious [security flaws or new areas for attack](https://www.drupal.org/docs/8/security)?

## General
- [ ] Is there anything included in this PR that is not related to the problem it is trying to solve?
- [ ] Is the approach to the problem appropriate?

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- JIRA ticket(s)
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# Resources
- [AMP Tool](https://stanford.levelaccess.net/index.php)
- [Accessibility Manual Test Script](https://docs.google.com/document/d/1ZXJ9RIUNXsS674ow9j3qJ2g1OAkCjmqMXl0Gs8XHEPQ/edit?usp=sharing)
- [HTML Validator](https://validator.w3.org/)
- [Browserstack](https://live.browserstack.com/dashboard) and link to [Browserstack Credentials](https://asconfluence.stanford.edu/confluence/display/SWS/External+Account+Credentials)

[AIS-46]: https://stanfordits.atlassian.net/browse/AIS-46?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: content/_components/composite-card.md
Comment (sherakama) [inline content/_components/composite-card.md:1]: I've got some bad news for you @jdwjdwjdw,

Most of the content in the content directory is generated from the inline comments and documentation in the https://github.com/SU-SWS/decanter/tree/v6 project. Try running `npm run build` on your local and see how much of this gets replaced... 

The generation script can be found here: https://github.com/SU-SWS/decanter-web/blob/decanter6/scripts/kss/generate.mjs

This script pulls in the `v6` branch, builds the KSS data from the SCSS files, and then renders markup using the twig files and default JSON data.

Long story short, you'll need to change the content in the `v6` branch on the `decanter` project to get at a number of these items. 

I don't really want anyone to be using v6 anymore so if you're up to retiring the v6 site instead of revising this work, then please go ahead.

Comment (jdwjdwjdw) [reply content/_components/composite-card.md:1]: Ahh okay, yeah I could tell something funky was going on, but was struggling to identifying why. Thanks for the heads up.

If that's the case, then I support your suggestion of just retiring the v6 site. I'll plan on removing all references to it on the decanter 7 site. I'm not entirely sure of the other steps necessary for retiring the v6 site, can you assist with letting me know what is needed there? Feel free to connect via slack or the related JIRA ticket as well.

Comment (sherakama) [reply content/_components/composite-card.md:1]: We use the v6 website in our preview builds on the decanter project so we will need to keep the codebase around, however, here is what we can do to retire the v6 site.

1. Remove all references from the V7 site. (Also remove references for support from the versions page)
2. Remove the decanter6.stanford.edu domain name from Netlify and delete the record from NetDB (I can help with the latter)
3. Put the site behind basic password protection (through Netlify's UI) and store the password in dashlane
4. We can sit on the password site for a bit before deleting the site out of Netlify.
